docs/data/stashcache/install-origin.md

```diff
@@ -0,0 +1,104 @@
+# StashCache Origin Installation Guide
+
+This document describes how to install a StashCache origin service. The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively. The role of the Origin and Cache is different, following explanation may help to decide when would you need to install which:
+
+* StashCache cache: role of **"cache"** server is to keep data cached and immediately available (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach) or CVMFS) within Stash federation (without re-transferring from "origin").
+* StashCache origin: The authoritative data server that hosts files locally and serves them to users upon transfer request (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach) or CVMFS) unless data are already cached.
+
+---
+
+## Installation prerequisites for Origin
+
+Before starting the installation process, consider the following mandatory points:
+
+* __User IDs:__ If they do not exist already, the installation will create the Linux user IDs condor and xrootd
+* __Host certificate:__ The StashCache server uses a host certificate to advertise to a central collector.  More information on how to receive a certificate can be found [here](/security/host-certs.md)
+* __Network ports:__ The StashCache service must listen on ports:
+    * XRootD service on port `1094 (TCP)`
+* __Hardware requirements:__ We recommend that a StashCache server has at least 10Gbps connectivity, 1TB of disk space, and 8GB of RAM. 
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Ensure the host has [a supported operating system](/release/supported_platforms.md)
+* Obtain root access to the host
+* Prepare [the required Yum repositories](/common/yum.md)
+* Install [CA certificates](/common/ca.md)
+
+## Installing the StashCache metapackage
+
+The StashCache daemon consists of an XRootD server and an HTCondor-based service for collecting and reporting statistics about the cache. To simplify installation, OSG provides convenience RPMs that install all required software with a single command:
+
+    [root@client ~]$ yum install stashcache-daemon fetch-crl stashcache-cache-origin
```

Comment:
See these sections: 

https://opensciencegrid.org/technology/documentation/style-guide/#command-blocks-and-file-snippets
https://opensciencegrid.org/technology/documentation/style-guide/#root-and-user-prompts